beast/physicsmodel/creategrid.py

```diff
@@ -737,84 +744,14 @@ def calc_absflux_cov_matrices(specgrid, sedgrid, filter_names):
     #     and fitting
     #   also convert from fractional to physical flux units
     m = 0
-    cov_diag[:,n_filters-1] = absflux_cov_mats[:,n_filters-1,n_filters-1]* \
-                              np.square(sedgrid.seds[:,n_filters-1])
+    cov_diag[:, n_filters-1] = (absflux_cov_mats[:, n_filters-1, n_filters-1]
+                                * np.square(sedgrid.seds[:, n_filters-1]))
     for k in range(n_filters-1):
-        cov_diag[:,k] = absflux_cov_mats[:,k,k]* \
-                        np.square(sedgrid.seds[:,k])
-        for l in range(k+1,n_filters):
-            cov_offdiag[:,m] = absflux_cov_mats[:,k,l]* \
-                               sedgrid.seds[:,k]*sedgrid.seds[:,l]
+        cov_diag[:, k] = (absflux_cov_mats[:, k, k]
+                          * np.square(sedgrid.seds[:, k]))
+        for l in range(k+1, n_filters):
+            cov_offdiag[:, m] = (absflux_cov_mats[:, k, l]
+                                 * sedgrid.seds[:, k]*sedgrid.seds[:, l])
             m += 1
 
     return (cov_diag, cov_offdiag)
-
-#=================== TESTUNITS ============================
```

Comment:
Are these useful/meaningful tests that should get moved over to test rather than deleted?

Reply:
Not sure.  These tests just checked to see if the code could be run.  What I would call a smoke test.  The regression tests that are done comparing to cached calculations are more comprehensive.  But they do require downloading data.  Maybe having such smoke tests would be a good idea.  Thoughts?  I will not merge these changes till we finish this conversation.  :-)

Reply:
One of the tests is was for old, deprecated code - so not needed.   The 2nd relied on a isochrone file that is no longer present.  It was not clear how to adapt that file without adding in the web download of the isochrone file, and this makes the test similar to the existing regression tests.  So, I don't think any of these tests are useful.  Would be nice to have simple tests that do not rely on downloading files, but I think that should be a separate issue.